src/abb_robot_client_RWS2/rws_aio.py

```diff
@@ -0,0 +1,892 @@
+# Copyright 2022 Wason Technology LLC, Rensselaer Polytechnic Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import struct
+import numpy as np
+import io
+import datetime
+import re
+import time
+import threading
+import asyncio
+
+from typing import Callable, NamedTuple, Any, List, Union
+import httpx
+import websockets
+
+from .rws import ABBException, RAPIDExecutionState, EventLogEntry, EventLogEntryEvent, TaskState, JointTarget, \
+    RobTarget, IpcMessage, Signal, ControllerState, OperationalMode, VariableValue, SubscriptionResourceType, \
+    SubscriptionResourcePriority, SubscriptionResourceRequest, SubscriptionException, SubscriptionClosed
+
+class RWS_AIO:

```

Comment:
Out of curiosity, are we expecting to update the asynchronous RWS commands in this PR? Or at all with the work we are scheduled to do? They original repo owner may not care about that functionality.